### PR TITLE
Add buffer library and inject into client package - Closes #5305

### LINKED ISSUE #5305: buffer.readBigUint64BE is not a function
### Expected behavior
Transaction and Cryptography library should be able to use in the front-end application

### Actual behavior
buffer.readBigUint64BE is used to convert buffer into bigint in `lisk-cryptography:bufferToIntAsString`. 
which is used to calculate transactionId in the `lisk-transactions` and `getAddressFromPublicKey` in `lisk-cryptography`.

Currently Frontend is using https://www.npmjs.com/package/buffer, but this library does not support the new `readBigUint64BE` and `writeBigUint64BE` functions

### Steps to reproduce
Try to create transaction in the frontend

### Which version(s) does this affect? (Environment, OS, etc...)
4.0-

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #5305 

### How was it solved?

Add browser version of Buffer which have bigint implementation temporally 

### How was it tested?

Open index.html in `elements/lisk-client/index.html` after build, and check function that uses bigint works in browser
```
lisk.cryptography.getAddressFromPassphrase('123')
```

### CODE REVIEW

File: elements/lisk-client/src/index.ts [hunk 1]
```diff
@@ -35,6 +42,7 @@ export default {
 	cryptography,
 	passphrase,
 	transactions,
+	Buffer,
 	// Also export as `transacation` for backward compatibility.
```
Comment (Incede) [inline elements/lisk-client/src/index.ts:46]: typo -> 'transacation'

**Review by reyraa — APPROVED** [review]
Looks good. Thanks

File: elements/lisk-client/index.html [hunk 2]
```diff
@@ -8,6 +8,6 @@
 		Open the console to get started with Lisk Client.
 		<div id="output"></div>
 
-		<script src="dist-browser/index.min.js"></script>
+		<script src="dist-browser/index.js"></script>
```
Comment (ManuGowda) [inline elements/lisk-client/index.html:11]: any specific reason for not using `min`?

**Review by ManuGowda — APPROVED** [review]

File: elements/lisk-client/index.html (hunk 2 above)
Comment (shuse2) [reply elements/lisk-client/index.html:11]: This file is for testing and i wanted to use unuglify version for test

**Review by Incede — APPROVED** [review]

**Pull request merged**